gordon/metrics/__init__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 Spotify AB
```

Comment:
Yeah the whole copyright thing confuses me a bit, as far as project vs. file vs. most recent file changes, but will fix!